client/levels/menu.tscn

```diff
@@ -40,6 +39,21 @@ region = Rect2(150, 50, 50, 50)
 atlas = ExtResource("7_1imcl")
 region = Rect2(500, 200, 50, 50)
 
+[sub_resource type="AtlasTexture" id="AtlasTexture_h8kge"]
+atlas = ExtResource("9_ekkjc")
+region = Rect2(480, 320, 32, 32)
+margin = Rect2(8, 0, 0, 0)
+
+[sub_resource type="GDScript" id="GDScript_ninfr"]
```

Comment:
will it make sense to create our own component, that simply have an export for the next scene?

Reply:
Not exactly sure what you mean. Maybe the code snipped is too small or wrong?

Reply:
no, my idea is to create the code snippet out, in a way that is easy reusable, without the path to the resource, but an exported variable. (but is only a thought)